### DISCUSSION TITLE
different result from whisper-timestamped and whisper

### DISCUSSION THREAD

**Discussion by Atefeh197** [linto-ai/whisper-timestamped#76] [General]
Hi

I try to do transcription with whisper-timestamped but the result is different with the whisper. 
Whisper-timestamped generates transcription with hallucination. 

Is it possible to get a result with Whisper-timestamped that is the same to whisper?

**Comment by Jeronymous** [comment]
The README explains this: default options are not the same in whisper and whisper-timestamped.
Look at the mention of those options: `beam_size=5, best_of=5, temperature=(0.0, 0.2, 0.4, 0.6, 0.8, 1.0)`
